src/modules/Core/Wallets/action.js

```diff
@@ -29,6 +32,9 @@ export const updateWalletsRequest = () => (dispatch: Dispatch, getState: GetStat
           const customTokens = state.ui.settings.customTokens
           const enabledNotHiddenTokens = enabledTokens.filter(token => {
             let isVisible = true // assume we will enable token
+            if (allSupportedParentCurrencies.includes(token)) {
+              return false
+            }
             const tokenIndex = _.findIndex(customTokens, item => item.currencyCode === token)
```

Comment:
There is no point in doing this filter here, since the next line, which searches for a custom token matching the currency code, will already eliminate these. We already prevent conflicting custom tokens from entering the system in the first place.